js/apps/admin-ui/src/realm-settings/NewClientPolicy.tsx

```diff
@@ -262,7 +262,7 @@ export default function NewClientPolicy() {
               toEditClientPolicy({ realm, policyName: formValues.name! }),
             );
           } catch (error) {
-            addError(t("deleteConditionError"), error);
+            addError("deleteConditionError", error);
```

Comment:
It seems that this label is actually missing, I cannot find it in any of the locales.